src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -312,38 +482,55 @@ void DoorShutter_SetupType(DoorShutter* this, PlayState* play) {
     }
 }
 
-f32 func_80996840(PlayState* play, DoorShutter* this, f32 arg2, f32 arg3, f32 arg4) {
+/**
+ * Compute the distance forwards to the player
+ * (perpendicularly to the door's left-right, and negative if the player is behind),
```

Comment:
`perpendicularly` sounds off (is it even a word?), I think perpendicularl to the door's left-right sounds fine

Reply:
yes it's a word and I think it's correctly used here but I can concede to perpendicular